code/game/dna/genes/vg_powers.dm

```diff
@@ -40,6 +40,7 @@ Obviously, requires DNA2.
 			M.update_mutations()		//update our mutation overlays
 			M.update_body()
 			to_chat(M, "<span class='warning'>You suddenly feel very weak.</span>")
+			M.species.muscle_mass = 150
```

Comment:
Does it make sense that hulking out sends you to magic numbers of muscle mass? e.g.: you workout and achieve 500 muscle mass, but now if you ever hulk out you'll drop back to 150